### PR TITLE
Allow overrides of the Firefox profile 

### PR DESCRIPTION
**Pull request by caffodian**
It is currently not possible to pass in a Firefox profile in Selenium, which means it is not possible to do things like setting download directory or install extensions.  

Probably ignore the extension case for now, but allow creating a profile and download directory.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -98,6 +100,18 @@ class SeleniumTestCase(RunTests):
...
+    label = 'Selenium'
+    test_app = 'nosedjangotests.selenium_autodownload_tests'
+    check_selenium = True
+    args = [
+        '--with-selenium',
+        '--with-django-sqlite',
+        '--ff-profile', 'browser.helperApps.neverAsk.saveToDisk="application/ourfakemimetype"',
+        '--ff-profile', 'browser.download.dir="{0}"'.format(tempfile.gettempdir()),
+        '--ff-profile', 'browser.download.folderList=2',
+        '--ff-profile', 'browser.download.manager.showWhenStarting=False'
```
Comment (caffodian) [inline setup.py]: argparse "*" would be nice here.  or maybe I am bad at optparse

File: nosedjango/plugins/selenium_plugin.py [hunk 2]
```diff
@@ -105,12 +116,25 @@ def get_driver(self):
             return self._driver
 
         if self._driver_type == 'firefox':
+            from selenium.webdriver.firefox.firefox_profile import FirefoxProfile  # noqa
+            fp = FirefoxProfile()
+
+            for override in self._ff_profile_overrides:
+                pref, value = override.split('=')
+                print value
```
Comment (winhamwr) [inline nosedjango/plugins/selenium_plugin.py]: Did you leave this print statement in on purpose?

Comment (caffodian) [inline nosedjango/plugins/selenium_plugin.py]: arghh

**Comment by winhamwr** [comment]
Throw in some basic documentation (in the README, prolly) and this looks good.

File: nosedjango/plugins/selenium_plugin.py [hunk 3]
```diff
@@ -65,18 +66,28 @@ def options(self, parser, env=None):
             ),
             default=None,
         )
+        parser.add_option(
+            '--ff-profile',
+            help=(
+                'Specify overrides for the FireFox profile'
+            ),
+            default=None,
```
Comment (kylegibson) [inline nosedjango/plugins/selenium_plugin.py]: This default causes problems below on line 122 :( `'NoneType' object is not iterable`

**Pull request merged**